migrate.sh

```diff
@@ -24,45 +24,50 @@ invenio index queue init purge
 # NOTE: db init is not needed since DB keeps being created
 #       Just need to create all tables from it.
 invenio db create
-# TODO: add back when pipenv access problem is fixed
-#invenio files location create --default 'default-location'  $(pipenv run invenio shell --no-term-title -c "print(app.instance_path)")'/data'
+# TODO: add back when pipenv access problem is fixed. Use this locally!!!
+# invenio files location create --default 'default-location'  $(pipenv run invenio shell --no-term-title -c "print(app.instance_path)")'/data'
```

Comment:
```suggestion
invenio files location create --default "default-location" $(invenio shell --no-term-title -c 'print(f"{app.instance_path}/data")'
```

This should also work, and then we can also uncomment the line completely and remove the one below. Checked on an OpenShift pod, and the output is the same:

```bash
[1011760000@web-debug src]$ invenio shell --no-term-title -c 'print(f"{app.instance_path}/data")'
/opt/invenio/var/instance/data
```

Reply:
Removed it!